### PR TITLE
Names with spaces: add string serializer

### PR DESCRIPTION
**Pull request by chapulina**
Addresses issue #239 .

This needs to target Dome because changing the serializer from default to a custom string serializer breaks ABI.

Added some names with spaces and special characters to `fuel.sdf` and everything seems to be in place:

names_spaces

### CODE REVIEW

**Comment by luca-della-vedova** [comment]
It did fix the errors being printed, however multiple models with spaces are still rendered incorrectly, i.e. the sdf snippet wrote in the issue renders as below:

image

File: examples/worlds/fuel.sdf [hunk 1]
```diff
@@ -122,6 +122,7 @@
 
     <!-- Included model without meshes -->
     <include>
+      <name>Double pendulum</name>
       <pose>-3 0 0 0 0 0</pose>
       <uri>https://fuel.ignitionrobotics.org/1.0/nate/models/double_pendulum_with_base/2</uri>
```
Comment (iche033) [inline examples/worlds/fuel.sdf:127]: I think @nkoenig removed some of these models from his account? I tried running this `fuel.sdf` world but fuel client complains about failing to download `double_pendulum_with_base` and `my_Backpack`, and ign-gazebo then exits.

Comment (chapulina) [reply examples/worlds/fuel.sdf:127]: Yeah those will be fixed when https://github.com/ignitionrobotics/ign-gazebo/pull/219 is merged forward.

**Comment by chapulina** [comment]
@osrf-jenkins retest this please

**Comment by chapulina** [comment]
* Ubuntu actions CI has never passed.
* Ubuntu Jenkins CI is currently broken for Dome.
* Homebrew doesn't have any new failures.

**Pull request merged**